pkg/certificate/mrc_reconciler.go

```diff
@@ -170,10 +180,42 @@ func (m *Manager) updateMRCState(mrc *v1alpha2.MeshRootCertificate) error {
 		mrc.Status.TransitionAfter = nil
 	} else {
 		mrc.Status.TransitionAfter = &metav1.Time{Time: time.Now().Add(mrcDurationPerStage)}
+		log.Debug().Str("mrc", namespacedMRCName(mrc)).Msgf("updated MRC TransitionAfter to %s", mrc.Status.TransitionAfter)
+	}
+
+	if shouldRetrieveCA(mrc) {
+		_, _, err := m.mrcClient.GetCertIssuerForMRC(mrc)
+		if err != nil {
+			log.Error().Err(err).Str("mrc", namespacedMRCName(mrc)).Msg("failed to retrieve CA on passive MRC creation")
+			if !m.leaderMode {
+				return err
+			}
+
+			// CA not accepted
+			setMRCCondition(mrc, constants.MRCConditionTypeAccepted, falseStatus, errorRetrievingCAReason, "")
```

Comment:
These messages should be set. In the case of an error, the error should be the message. Else, the message should be a more human readable description of why the condition is what it is

Reply:
Updated. I might define the messages as constants with the other condition variables https://github.com/openservicemesh/osm/pull/5183#discussion_r985827964